chore: create `TokenCreator` role with permissions to VIEW / CREATE / REVOKE own access tokens (#10064)
determined-ai/determined: docs/release-notes/rbac-new-tokenCreator-role.rst

:orphan:

**New Features**

> -  New RBAC role: Added a new ``TokenCreator`` RBAC role. It grants the permissions to create, view,

Reviewer: ```suggestion
- New RBAC role: Add a ``TokenCreator`` RBAC role, which allows users to create, view,
```

>    and revoke the user's own access tokens. It can only be assigned globally.

Reviewer: ```suggestion
and revoke their own access tokens. This role can only be assigned globally.
```
